decisions/wc_lifestyle_decisions.txt

```diff
@@ -0,0 +1,242 @@
+decisions = {
+	become_brewmaster = {
+		is_high_prio = yes # Will show an alert once decision is enabled
+		
+		ai_check_interval = 120
+		
+		only_playable = yes
+		
+		potential = {
+			#Brewmaster race or player
+			OR = {
+				trait = creature_vrykul
+				trait = creature_ogre
+				trait = creature_dwarf
+				ai = no
+			}
+			has_focus = focus_carousing
+			lifestyle_traits = 0
+			NOT = { has_character_flag = becoming_brewmaster }
+			NOT = {	trait = brewmaster }
+		}
+		allow = {
+			custom_tooltip = {
+				text = is_not_busy_trigger_tooltip
+				hidden_tooltip = { NOT = { has_character_flag = do_not_disturb } }
+			}
+			wealth = 100
+			lifestyle_traits < 1
+		}
+		effect = {
+			set_character_flag = do_not_disturb
+			set_character_flag = becoming_brewmaster
+			wealth = -100
+			character_event = { id = WCLIFES.1 }
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+	make_brew = {
+		is_high_prio = yes # Will show an alert once decision is enabled
+		
+		ai_check_interval = 120
+		
+		only_playable = yes
+		
+		potential = {
+			trait = brewmaster
+		}
+		allow = {
+			custom_tooltip = {
+				text = is_not_busy_trigger_tooltip
+				hidden_tooltip = { NOT = { has_character_flag = do_not_disturb } }
+			}
+			wealth = 100
+		}
+		effect = {
+			set_character_flag = do_not_disturb
+			wealth = -100
+			character_event = { id = WCLIFES.10 }
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+}
+targetted_decisions = {
+	drink_brew = {
+		filter = realm_including_me
+		ai_target_filter = court_including_me
+		ai_check_interval = 24
+
+		only_playable = yes
+
+		from_potential = {
+			is_untouchable_trigger = no
+
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			
+			trait = brewmaster
+		}
+
+		potential = {
+			is_untouchable_trigger = no
+			
+			is_adult = yes
+			
+			is_within_diplo_range = FROM
+		}
```

Comment:
So, you can just force a drink down to someone's else throat?

<details>
<summary>Even if they can't drink at all</summary>

![image](https://user-images.githubusercontent.com/46576860/91623783-906a9080-e9a5-11ea-8699-d766ee94f3f7.png)
![image](https://user-images.githubusercontent.com/46576860/91623797-a37d6080-e9a5-11ea-9de3-0b38c462c90c.png)

</details>